src/product-info.js

```diff
@@ -47,11 +47,11 @@ export default {
     primaryTask: 'Design APIs',
     description: 'The Apicurito RESTful API visual designer.'
   },
-  mdc: {
-    prettyName: 'Mobile Developer Console',
-    gaStatus: 'preview',
-    primaryTask: 'Create mobile apps',
-    description: 'Allows you register your mobile app and associate it with Mobile Developer Services.'
+  unifiedpush: {
+    prettyName: 'Push Notification Service',
+    gaStatus: 'GA',
+    primaryTask: 'Mobile Push Messaging',
+    description: 'Provides APNS and FCM messaging services for mobile applications.'
```

Comment:
Can we make this description more user friendly? I'm not sure general users would know what APNS or FCM mean. Happy tow work with you on this.